docs/doxygen/overviews/archive.h

```diff
@@ -179,27 +181,22 @@ entries looked up by name can be opened using the
 wxArchiveInputStream::OpenEntry() method.
 
 @code
-WX_DECLARE_STRING_HASH_MAP(wxZipEntry*, ZipCatalog);
-ZipCatalog::iterator it;
-wxZipEntry *entry;
+using ZipEntryPtr = std::shared_ptr<wxZipEntry>;
```

Comment:
Do we really need to use `shared_`, rather than `unique_`, pointer here? Not a problem if we do, I just don't see why should we.

Reply:
Fixed.